Geometry_Engine/Create/ShapeProfileCurves.cs

```diff
@@ -34,32 +34,36 @@ public static partial class Create
         /**** Public Methods                            ****/
         /***************************************************/
 
-        public static List<ICurve> IProfileCurves(double tft, double tfw, double bft, double bfw, double wt, double wd, double r1, double r2)
+        public static List<ICurve> IProfileCurves(double tft, double tfw, double bft, double bfw, double wt, double wd, double r1, double r2, double weldSize)
```

Comment:
A bit scary to allow both root radius and weld size, but as this mostly is an internal method being called indirectly it could be fine. Could be good if you could have a quick look at adding warning or splitting up in two different methods though I think.